leetcode/src/61.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * Given the head of a linked list, rotate the list to the right by k places.
+ *
+ *
+ * Definition for singly-linked list.
+ * struct ListNode {
+ *     int val;
+ *     struct ListNode *next;
+ * };
+ */
+struct ListNode* rotateRight(struct ListNode* head, int k){
+    struct ListNode *ptr = NULL, *prev =NULL;
+    ptr = head;
+    int count = 0;
+    while(ptr){
+        ptr = ptr->next;
+        count++;
+    }
```

Comment:
This is more readable than `if(ptr==NULL)
            return;
        if(ptr->next==NULL)
            return ptr;` and can handle `count == 0` as well, removing the need for a separate `count != 0` check
```suggestion
    }
    if(count < 2) {
        return NULL;
    }
```